core/src/test/java/jenkins/util/TreeStringBuilderTest.java

```diff
@@ -12,8 +12,6 @@
  *
  * @author Kohsuke Kawaguchi
  */
-@SuppressWarnings({"PMD", "all"})
-//CHECKSTYLE:OFF
```

Comment:
This file complies with our Checkstyle checks, so I see no reason to turn Checkstyle off.